src/wp-admin/css/dashboard.css

```diff
@@ -1,1316 +1,1418 @@
 #wpbody-content #dashboard-widgets.columns-1 .postbox-container {
-	width: 100%;
```

Comment:
You probably need to use tabs instead of spaces for line indentions.

Reply:
Yes, there are multiple files overwritten / reformatted here.  Possibly a broken editor configuration.

Reply:
I have a linter which I need to turn off

Reply:
I realized from GitHub, that it uses 8 spaces for tab. I am adjusting that in my editor.